### ISSUE TITLE
Feature suggestions - Copy image / prompt on right click

### ISSUE THREAD

**Issue by Aamir3d** [RupertAvery/DiffusionToolkit#221]
Current right click functionality does a "Fit to Preview"

A better use of this would be to either give an option for shortcuts, or even better, have a right click menu that allows a user to copy either the prompt or the image or the image path directly.

1. Copy Prompt - useful when copy pasting fast rather than having to go to the info bar and click copy
2. Copy Image - useful to paste into other image programs for quick comparison or resizing/editing
3. Copy Image path - useful when pasting an image path into applications for uploading / actioning

Some of this functionality is currently available in the Thumbnail > Right click options, but 'copy' doesn't seem to do anything there (or 
I'm mistaken. If I right click > copy, or press Ctrl+c, nothing happens when pasting).

**Comment by RupertAvery** [comment]
Yes, this shoukd be easy

**Comment by RupertAvery** [comment]
Yes, should be the same name as the one in my Github profile. Thanks!

**Comment by Aamir3d** [comment]
![image](https://github.com/RupertAvery/DiffusionToolkit/assets/122131887/29697b7f-5a04-472b-a8dd-83a3f3dca2d7)
I don't see it here. Can we connect ? https://www.linkedin.com/in/aamir-kadri/
